### ISSUE TITLE
Gallery Block: Support Grouping

### ISSUE THREAD

**Issue by archetyped** [archetyped/simple-lightbox#802]
## Description

Support grouping of images in gallery blocks added via WordPress' block editor (Gutenberg) when the option is enabled in SLB's admin settings.

**Comment by neiltking** [comment]
I thought ticking the "Group gallery items separately" option would make each WordPress gallery on the page have its own group of images for the lightbox/slideshow. This is not the case as it shows all images from all galleries on the page.

**Comment by archetyped** [comment]
@neiltking please clarify: are you referring to gallery grouping when using the block editor (which the original post already covers) or the classic editor?

If you are experiencing an issue with gallery grouping with the classic editor, please create a new ticket so that your specific issue can be fully evaluated.

Thank you.

**Comment by neiltking** [comment]
@archetyped Yes - this is with the Gutenberg block editor. My apologies of my comment just duplicated the information in the original ticket but I wasn't 100% sure it described the same issue. Thanks for looking into this.

**Comment by sheblair** [comment]
Any updates on this feature?
